crates/bevy_ecs/src/query/state.rs

```diff
@@ -61,6 +62,9 @@ impl<Q: WorldQuery, F: WorldQuery> QueryState<Q, F> {
 
         let mut state = Self {
             world_id: world.id(),
+            task_pool: world
+                .get_resource::<ComputeTaskPool>()
+                .map(|task_pool| task_pool.deref().clone()),
```

Comment:
The clone still appears to be present.

Reply:
This clone is unavoidable. The one referred to above is inside `par_for_each_unchecked_manual`.